gax-grpc/build.gradle

```diff
@@ -13,18 +13,16 @@ dependencies {
     libraries.authCredentials,
     libraries.commonProtos,
     libraries.apiCommon,
-    libraries.grpcNettyShaded
+    libraries.grpcNetty
```

Comment:
The dependency should be `grpcNettyShaded`...

Reply:
They were originally named grpcNetty even it was shaded: https://github.com/googleapis/gax-java/blob/3e64fc5fe03eb3f0730fc73e050c12b32a4e2cc0/build.gradle#L115

Reply:
oh! makes sense now.